tests/operator_tests.py

```diff
@@ -294,6 +294,10 @@ def testConcatenate(self):
             ak.concatenate([pdaOne, pdaTwo]).to_list(),
         )
 
+    def test_invert(self):
+        ak_uint = ak.arange(10, dtype=ak.uint64)
+        self.assertListEqual(np.arange(10, dtype=np.uint).tolist(), ak_uint.to_list())
```

Comment:
I don't think we're actually calling `~` here

Reply:
My fault. I copied my test out of `ipython` incorrectly. Should be corrected.